suggestions/calendars.json

```diff
@@ -4,5 +4,11 @@
         "label": "Etar",
         "source": "fg",
         "repo": "https://github.com/Etar-Group/Etar-Calendar"
+    },
+    {
+        "id": "com.simplemobiletools.calendar.pro",
+        "label": "Simple Calendar Pro",
+        "source": "fg",
+        "repo": "https://github.com/SimpleMobileTools/Simple-Calendar"
```

Comment:
Requires a review to keep only one out of the two. @shuvashish76: Opinions?

Reply:
Etar supports multiple offline calendars though you don't get as frequent updates as SCP. SCP paid on playstore vs Etar free. Both project active & stable enough for daily use. SCP is their 2nd popular (_based on GH stars_) app from SMT, so I know a lots of users use it. Need to use it for longer time for proper review. Give us your review @DodoLeDev

Reply:
Initial comparison table

| Feature | SCP | Etar
| --- | --- | ---
| Material Design 3 | ✅ | ❌
| Add calendar events | ✅ (events and tasks) | ✅
| CalDav sync | ✅ (opt-out) | ✅ (opt-in)
| Widgets | ✅ | ✅ (only one widget)
| Offline calendars | ✅ (via event types) | ✅
| Custom day for starting a week | ❌ (only western) | ✅ (global)
| Automatic backups | ✅ | ❌
| Print calendar | ✅ | ❌
| Subscription | Free on F-Droid and premium on Play Store | Free
| Size (on F-Droid) | 9.4 MB | 6.2 MB
| Updates | Frequent updates | Less updates
| Maintainability | Backed by Simple Mobile Tools | Backed by Etar and Lineage OS

SCP gains more point than Etar.

Reply:
> https://github.com/Etar-Group/Etar-Calendar#technical-explanation

Etar calendars are visible to other apps e.g [Tasks](https://github.com/tasks/tasks), no such feature for SCP. SCP events visible to that app only.

> SCP gains more points than Etar.

Agreed 👍

Reply:
> Etar calendars are visible to other apps e.g [Tasks](https://github.com/tasks/tasks), no such feature for SCP. SCP events visible to that app only.

This wasn't considered because this is not something the app should do. It should be managed by a separate calendar provider (e.g. DecSync). Calendar, contacts, etc. in Android is divided into providers and clients for giving users the freedom to choose their favourite clients, and ideally, these clients should never be installed in the system partition (should be installed as user apps during initial setup). There was such a minimal OS called Magma Moonshine (only 4/5 launcher apps) which, unfortunately, was discontinued due to lack of time.

Reply:
So... Should we keep both apps, or remove Etar from the list?

Reply:
> So... Should we keep both apps, or remove Etar from the list?

I think Etar should be removed in favour of SCP. Etar has some good features but it's still less appealing than SCP in terms of features that a typical user's looking for.